agent/config/parse_windows.go

```diff
@@ -34,14 +37,30 @@ const (
 // parseGMSACapability is used to determine if gMSA support can be enabled
 func parseGMSACapability() bool {
 	envStatus := utils.ParseBool(os.Getenv("ECS_GMSA_SUPPORTED"), true)
+	return checkDomainJoinWithEnvOverride(envStatus)
+}
+
+// parseFSxWindowsFileServerCapability is used to determine if fsxWindowsFileServer support can be enabled
+func parseFSxWindowsFileServerCapability() bool {
+	// fsxwindowsfileserver is not supported on Windows 2016 and non-domain-joined container instances
+	status, err := isWindows2016()
+	if err != nil || status == true {
```

Comment:
Shouldn't this be `status == false`?

Reply:
I think this should be ```status==true``` which implies that capability will not be advertised if its a windows 2016 container instance.

Reply:
Yes, probably worth adding a comment to indicate that this will not work on WS2016

Reply:
yes, I agree. I added a comment right above on L45 👍